### DISCUSSION TITLE
Love the new update, but ...

### DISCUSSION THREAD

**Discussion by dgcasey0325** [SauceLLC/sauce4zwift#100] [General]
Just a small problem with the new Watching windows. Love the new groups that can be added to the Watching windows AND the ability to change the title one each tile. However, it appears that changing the title on an Event tile won't take effect. Other than that I like what I see.
![Image1](https://github.com/SauceLLC/sauce4zwift/assets/38332950/24b51754-3f6b-4714-ba90-f2aca9f11dcb)

**Comment by mayfield** [comment]
Hi @dgcasey0325 sounds like a bug, I moved this to issue #101
